src/NodeParameters.cpp

```diff
@@ -26,6 +26,9 @@ void NodeParameters::declareParameters(rclcpp::Node& node)
     node.declare_parameter<bool>("is_online", true);
     node.declare_parameter<bool>("save_map_cells_on_hard_drive", true);
     node.declare_parameter<bool>("publish_tfs_between_registrations", true);
+    node.declare_parameter<bool>("deskew", true);
+    node.declare_parameter<int>("expectedUniqueDeskewingTFNumber", 4000);
```

Comment:
The new parameter names should be in snake_case to follow the current convention